runelite-client/src/main/java/net/runelite/client/plugins/groundmarkers/GroundMarkerOverlay.java

```diff
@@ -42,6 +42,8 @@
 
 public class GroundMarkerOverlay extends Overlay
 {
+	private static final int MAX_DRAW_DISTANCE = 32;
```

Comment:
I am pretty sure we have max distance defined in constants, so if you are already changing this, you should maybe use that.

Reply:
Doesn't look like it. [Constants](https://github.com/runelite/runelite/blob/master/runelite-api/src/main/java/net/runelite/api/Constants.java) doesn't have anything like this, and #4157 hasn't been closed so I don't think that's the case.

Reply:
Ah yeah I guess I never PRed that after the agility plugin revert